src/serac/physics/elasticity.hpp

```diff
@@ -18,6 +18,16 @@
 
 namespace serac {
 
+class Elasticity;
+
+namespace detail {
```

Comment:
👍 @joshessman-llnl @white238 , does this look OK to you?

Reply:
After speaking with @jamiebramwell .  Exposing this is okay as long as we make it very clear you can shoot yourself in the foot.  Confusing the interface is not adding anything.

We should just expose what we need to and put a note in the documentation for the buyer-beware nature...  Please reword to be correct.

```
/**
  Get the current gradient MFEM operator

  Note:  This is for expert users only, changing any values inside of the returned data structures can have drastic and unrecoverable runtime consequences.
**/
const mfem::operator& current_gradient();
```

Reply:
Sorry for flip-flopping! After kicking it around in my head, I agree with @white238 that returning a `const mfem::Operator&` is safe enough and it is clear what is going on.